SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -35,6 +37,13 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 public:
 	virtual void FinishDestroy() override;
 	void DestroyConnection();
+	
+	using LoginTokenRes_Callback = TFunction<bool(const Worker_Alpha_LoginTokensResponse*)>;
```

Comment:
Unreal does not use _ in it's naming convention.

Reply:
But I see there are some names using '_'.  That is confuse for me.
```
Worker_Connection* WorkerConnection;
Worker_Locator* WorkerLocator;
Worker_RequestId NextRequestId = 0;
```

Reply:
Yes that is a little confusing. This is because Worker_Connection is part of the Worker SDK. This exist outside of the Unreal GDK but we use it in our code. It does not have to meet the same standard.

Reply:
It is better to unify code style. I have saw three different code style in improbable.

Reply:
Possibly but this would now be a huge task to unify all teams. It is understandable that there are different styles. I imagine the Worker SDK was created before the Unreal GDK so there is no way the Worker SDK should have taken it's standard from unreal. When writing the Unreal GDK it made sense to use the Unreal standards as we are rewriting a lot of Unreal engine code. In an ideal world unifying would be great.